[2023-09-24T11:04:39.039+0000] {logging_mixin.py:151} INFO - Changing /opt/***/logs/dag_id=dag_with_postgres_operator_v02/run_id=scheduled__2023-09-19T00:00:00+00:00/task_id=create_postgres_table permission to 509
[2023-09-24T11:04:39.049+0000] {logging_mixin.py:151} INFO - Failed to change /opt/***/logs/dag_id=dag_with_postgres_operator_v02/run_id=scheduled__2023-09-19T00:00:00+00:00/task_id=create_postgres_table permission to 509: [Errno 1] Operation not permitted: '/opt/***/logs/dag_id=dag_with_postgres_operator_v02/run_id=scheduled__2023-09-19T00:00:00+00:00/task_id=create_postgres_table'
[2023-09-24T11:04:39.158+0000] {logging_mixin.py:151} INFO - Changing /opt/***/logs/dag_id=dag_with_postgres_operator_v02/run_id=scheduled__2023-09-19T00:00:00+00:00/task_id=create_postgres_table permission to 509
[2023-09-24T11:04:39.160+0000] {logging_mixin.py:151} INFO - Failed to change /opt/***/logs/dag_id=dag_with_postgres_operator_v02/run_id=scheduled__2023-09-19T00:00:00+00:00/task_id=create_postgres_table permission to 509: [Errno 1] Operation not permitted: '/opt/***/logs/dag_id=dag_with_postgres_operator_v02/run_id=scheduled__2023-09-19T00:00:00+00:00/task_id=create_postgres_table'
[2023-09-24T11:04:39.296+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: dag_with_postgres_operator_v02.create_postgres_table scheduled__2023-09-19T00:00:00+00:00 [queued]>
[2023-09-24T11:04:39.321+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: dag_with_postgres_operator_v02.create_postgres_table scheduled__2023-09-19T00:00:00+00:00 [queued]>
[2023-09-24T11:04:39.325+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 6
[2023-09-24T11:04:39.444+0000] {taskinstance.py:1382} INFO - Executing <Task(PostgresOperator): create_postgres_table> on 2023-09-19 00:00:00+00:00
[2023-09-24T11:04:39.480+0000] {standard_task_runner.py:57} INFO - Started process 194 to run task
[2023-09-24T11:04:39.524+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'dag_with_postgres_operator_v02', 'create_postgres_table', 'scheduled__2023-09-19T00:00:00+00:00', '--job-id', '118', '--raw', '--subdir', 'DAGS_FOLDER/dag-postgres-operator.py', '--cfg-path', '/tmp/tmp_nhu5do0']
[2023-09-24T11:04:39.547+0000] {standard_task_runner.py:85} INFO - Job 118: Subtask create_postgres_table
[2023-09-24T11:04:40.338+0000] {logging_mixin.py:151} INFO - Changing /opt/***/logs/dag_id=dag_with_postgres_operator_v02/run_id=scheduled__2023-09-19T00:00:00+00:00/task_id=create_postgres_table permission to 509
[2023-09-24T11:04:40.362+0000] {logging_mixin.py:151} INFO - Failed to change /opt/***/logs/dag_id=dag_with_postgres_operator_v02/run_id=scheduled__2023-09-19T00:00:00+00:00/task_id=create_postgres_table permission to 509: [Errno 1] Operation not permitted: '/opt/***/logs/dag_id=dag_with_postgres_operator_v02/run_id=scheduled__2023-09-19T00:00:00+00:00/task_id=create_postgres_table'
[2023-09-24T11:04:40.379+0000] {task_command.py:415} INFO - Running <TaskInstance: dag_with_postgres_operator_v02.create_postgres_table scheduled__2023-09-19T00:00:00+00:00 [running]> on host 6c82fec646db
[2023-09-24T11:04:41.254+0000] {taskinstance.py:1660} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='qpsss' AIRFLOW_CTX_DAG_ID='dag_with_postgres_operator_v02' AIRFLOW_CTX_TASK_ID='create_postgres_table' AIRFLOW_CTX_EXECUTION_DATE='2023-09-19T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2023-09-19T00:00:00+00:00'
[2023-09-24T11:04:41.265+0000] {sql.py:274} INFO - Executing: 
            create table if not exists dag_runs (
                dt date,
                dag_id character varying,
                primary key (dt, dag_id)
            )
        
[2023-09-24T11:04:41.315+0000] {base.py:73} INFO - Using connection ID 'postgres_localhost' for task execution.
[2023-09-24T11:04:41.388+0000] {base.py:73} INFO - Using connection ID 'postgres_localhost' for task execution.
[2023-09-24T11:04:41.517+0000] {sql.py:418} INFO - Running statement: 
            create table if not exists dag_runs (
                dt date,
                dag_id character varying,
                primary key (dt, dag_id)
            )
        , parameters: None
[2023-09-24T11:04:41.572+0000] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=dag_with_postgres_operator_v02, task_id=create_postgres_table, execution_date=20230919T000000, start_date=20230924T110439, end_date=20230924T110441
[2023-09-24T11:04:41.802+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2023-09-24T11:04:42.088+0000] {taskinstance.py:2784} INFO - 0 downstream tasks scheduled from follow-on schedule check
